spec/unit/shoulda/matchers/action_controller/callback_matcher_spec.rb

```diff
@@ -47,10 +47,90 @@ def match
       end
     end
 
+    context "qualified with only" do
```

Comment:
Prefer single-quoted strings when you don't need string interpolation or special symbols.